CMakeLists.txt

```diff
@@ -194,9 +195,15 @@ if(UNIX AND APPLE)
     set(MACOSX_BUNDLE_LONG_VERSION_STRING "${Launcher_VERSION_MAJOR}.${Launcher_VERSION_MINOR}.${Launcher_VERSION_HOTFIX}")
     set(MACOSX_BUNDLE_ICON_FILE ${Launcher_Name}.icns)
     set(MACOSX_BUNDLE_COPYRIGHT "Copyright 2021-2022 ${Launcher_Copyright}")
+    set(MACOSX_SPARKLE_UPDATE_PUBLIC_KEY "qATG/94DDm31fqxgoAEl0wbfVB3fWiL3XQP1YH9LWTA=")
+    set(MACOSX_SPARKLE_UPDATE_FEED_URL "https://gist.githubusercontent.com/kthchew/a88603ad35aef8ada84d574c74bd5a4e/raw/test-appcast.xml")
```

Comment:
Is it maybe possible to somehow use GitHub releases instead?

Reply:
Do you mean instead of using an RSS feed? Unfortunately, I think it only uses appcasts. There’s actually an open Sparkle feature request for this very thing: https://github.com/sparkle-project/Sparkle/issues/648. The closest you can get to that right now is automatically generating the appcast when a new release is made.

Reply:
Hmm. We would need tooling for that. Maybe we can hack something with GitHub Actions and running a Pages instance for this repo.

Reply:
There's already a separate GitHub pages repo that contains an RSS feed for news, right? I remember @ZekeSmith saying it might be able to be adapted to implement this. If it's possible to add the [necessary tags](https://github.com/PolyMC/PolyMC/pull/479#issuecomment-1124516056) to that feed for release news, then it would probably work.

Reply:
Given that the CI uses a macOS machine to build the macOS version, it is also possible to use Sparkle's automatic tool to make the appcast (it's included in the release archive that I had CMake download, under `bin/generate_appcast`). Although some stuff, like the download URL, may need to be edited after it generates the appcast, but that should be fairly easy.